rocky/katalogus/templates/katalogus_settings.html

```diff
@@ -44,13 +44,11 @@ <h1>{% translate "KAT-alogus Settings" %}</h1>
                     </div>
                 </div>
```

Comment:
This div inside the else clause needs to be removed, not the one outside. If this is fixed the endif should also be indented on the same level as the if/else.

Reply:
Good catch. Will fix this